cypress/integration/fee-fine/fee-fine-history.spec.js

```diff
@@ -0,0 +1,139 @@
+import moment from 'moment';
+import permissions from '../../support/dictionary/permissions';
+import TestTypes from '../../support/dictionary/testTypes';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import Users from '../../support/fragments/users/users';
+import UsersOwners, { getNewOwner } from '../../support/fragments/settings/users/usersOwners';
+import TransferAccounts, { getNewTransferAccount } from '../../support/fragments/settings/users/transferAccounts';
+import ManualCharges from '../../support/fragments/settings/users/manualCharges';
+import PaymentMethods from '../../support/fragments/settings/users/paymentMethods';
+import WaiveReasons, { getNewWaiveReason } from '../../support/fragments/settings/users/waiveReasons';
+import RefundReasons, { getNewRefundReason } from '../../support/fragments/settings/users/refundReasons';
+import NewFeeFine from '../../support/fragments/users/newFeeFine';
+import AppPaths from '../../support/fragments/app-paths';
+import FeeFinesDetails from '../../support/fragments/users/feeFineDetails';
+import PayFeeFaine from '../../support/fragments/users/payFeeFaine';
+import AddNewStaffInfo from '../../support/fragments/users/addNewStaffInfo';
+import { HTML } from '../../../interactors';
+
+//The test fails because of the bug.
```

Comment:
fix linter issue

Reply:
I need this comment because it explains the reason why the test fails.

Reply:
Done.